prusti-launch/src/bin/prusti-rustc.rs

```diff
@@ -52,10 +52,12 @@ fn process(mut args: Vec<String>) -> Result<(), i32> {
 
     // Setting RUSTC_WRAPPER causes Cargo to pass 'rustc' as the first argument.
     // We're invoking the compiler programmatically, so we ignore this
-    let cargo_invoked =
-        !args.is_empty() && Path::new(&args[0]).file_stem() == Some("rustc".as_ref());
-    if cargo_invoked {
-        args.remove(0);
+    let rustc_pos = args.iter().position(|arg|
+        Path::new(arg).file_stem() == Some("rustc".as_ref())
+    );
+    let cargo_invoked = rustc_pos.is_some();
+    if let Some(rustc_pos) = rustc_pos {
+        args.drain(0..=rustc_pos);
```

Comment:
What do the dropped arguments look like?

Reply:
You can see in the failing run (https://github.com/viperproject/prusti-dev/runs/7939531751?check_suite_focus=true): `/home/runner/work/prusti-dev/prusti-dev/target/debug/prusti-rustc /home/runner/.rustup/toolchains/nightly-2022-08-01-x86_64-unknown-linux-gnu/bin/clippy-driver rustc - --crate-name ___ --print=file-names --crate-type bin --crate-type rlib --crate-type dylib --crate-type cdylib --crate-type staticlib --crate-type proc-macro --print=sysroot --print=cfg`

Reply:
So `/home/runner/.rustup/toolchains/nightly-2022-08-01-x86_64-unknown-linux-gnu/bin/clippy-driver rustc` gets dropped